### ISSUE TITLE
Broken NAV on /get-almalinux page

### ISSUE THREAD

**Issue by codyro** [AlmaLinux/almalinux.org#318]
https://github.com/AlmaLinux/almalinux.org/pull/254 introduced a new page, https://almalinux.org/get-almalinux/. On this page, a NAV appears to be broken and behaves "wonkily". See the attached video for reference:

https://github.com/AlmaLinux/almalinux.org/assets/196810/7bff80c9-712e-4128-bc5d-81fa4abdc133

**Comment by codyro** [comment]
Yes, also the last two items (IBM Z/Power LE) seem to have some image padding issues as they're displaying the same images, but the images display differently. Click them back and forth (or it's near the end of the video) to see what I mean.

**Comment by bennyvasquez** [comment]
@sboldyreva fixed the images problem in #321, and will take a look at the rest. 

In talking with her it seems the double-clicking is working as intended. It's like a dropdown menu, so the "menu" collapses on second click. I think, from an end user perspective, this isn't how I would expect this content to work. I see two potential fixes:

1. disable the collapsing of the data on a second click
2. add a small arrow next to the menu item as an indicator of expanded or collapsed information. 

The first option is my preference, but the second can be a short-term fix if 1 is unattainable in short order.

**Comment by sboldyreva** [comment]
@codyro @bennyvasquez Yes, I confirm this. And I agree, we might want to disable collapsing, to keep the content open.

**Comment by sboldyreva** [comment]
As for the icon images (s390x and ppc64le) they should be displayed the same after the fix.

**Comment by codyro** [comment]
I think disabling collapsing would make sense from a UX perspective.

**Comment by bennyvasquez** [comment]
Assigning this to you, @sboldyreva, but only so that my brain stops bothering me about it. Let me know if you need an assist here!
